rpc/src/lib.rs

```diff
@@ -10,8 +10,13 @@
 //!
 //! | Feature | Description |
 //! | ------- | ----------- |
-//! | `http-client` | Provides [`HttpClient`], which is a basic RPC client that interacts with remote Tendermint nodes via **JSON-RPC over HTTP**. This client does not provide [`Event`] subscription functionality. See the [Tendermint RPC] for more details. |
-//! | `websocket-client` | Provides [`WebSocketClient`], which currently only provides [`Event`] subscription functionality over a WebSocket connection. See the [`/subscribe` endpoint] in the Tendermint RPC for more details. This client does not yet provide access to the RPC methods provided by the [`Client`] trait (this is planned for a future release). |
+//! | `http-client` | Provides [`HttpClient`], which is a basic RPC client that interacts with
```

Comment:
Does this still render correctly when wrapped? I don't think it should be wrapped.

Reply:
must have been fmt or me adding a typo. Fixed in the next commit.

Reply:
Did this get fixed? Looks like it's still wrapping?